app/assets/stylesheets/application.css.scss

```diff
@@ -69,6 +69,18 @@ body {
 
 .application-body-bree{
   background-color: black;
+  background-image: url('bree.jpg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-rivendell{
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.